### PR TITLE
Remove uplc-genvironment-instance.uplc from kplutus-files.

### PR DESCRIPTION
**Pull request by ChristianoBraga**
- Remove `uplc-genvironment-instance.uplc` from `kplc-files` in the Makefile. Add it to `tests/spec/simple`. Each verificable UPLC code must have `verification` dir, `verification.md` and possibly `uplc-genvironment-instance.md` file.`
- Add `.expected` files to `tests/textual/benchmark-validation-examples`.
- Update `tests/failing` by removing filenames in `tests/textual/benchmark-validation-examples`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -243,6 +242,7 @@ kplutus_files := uplc.md \
                  uplc-discharge.md \
                  uplc-free-variables.md \
                  uplc-abstract-environment.md
+#                 uplc-genvironment-instance.md \
```
Comment (gtrepta) [inline Makefile:245]:
```suggestion
```

File: tests/failing [hunk 2]
```diff
@@ -31,92 +31,14 @@ tests/textual/uplc-examples/DivideByZeroDrop.uplc
...
-tests/textual/benchmark-validation-examples/vesting-1.uplc
 #
 # Nofib-exe examples
 #
 tests/textual/nofib-exe-examples/clausify-F7.uplc
 tests/textual/nofib-exe-examples/knights-500-500.uplc
 tests/textual/nofib-exe-examples/lastpiece.uplc
 tests/textual/nofib-exe-examples/prime-P200.uplc
-tests/textual/nofib-exe-examples/primetest-large.uplc
+# tests/textual/nofib-exe-examples/primetest-large.uplc
```
Comment (gtrepta) [inline tests/failing:41]:
```suggestion
```

File: .github/workflows/test-pr.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ concurrency:
 jobs:
 
   test-pr:
-    name: 'Bulid and test KPlutus'
+    name: 'Build and test KPlutus'
```
Comment (ehildenb) [inline .github/workflows/test-pr.yml:13]: You should do this change elsewhere, or I can adjust the check name name. But this affects the required status checks.

Comment (ehildenb) [reply .github/workflows/test-pr.yml:13]: Can leave this change for my PR: https://github.com/runtimeverification/plutus-core-semantics/pull/359

**Review by gtrepta — APPROVED** [review]

**Pull request merged**